CHANGELOG.latest.md

```diff
@@ -1,9 +1,9 @@
+### Breaking Changes
+* Support for Billing Client 6 (#1209) via Mark Villacampa (@MarkVillacampa)
```

Comment:
Do we want to write this up better and maybe add a link to the docs or migration guide?

Also might be good to write up a bit better the other items on the changelog.